test/index.html

```diff
@@ -21,10 +21,10 @@
   <body>
     <script>
       WCT.loadSuites([
-        // Load 'my-view1.html' test suite, using native shadow dom
         'my-view1.html?dom=shadow',
-        // Load 'my-view1.html' test suite, using shadydom
-        'my-view1.html?wc-shadydom=true',
+        'ryr-location.html?wc-shadydom=true',
+        'my-view1.html?dom=shadow',
+        'ryr-location.html?wc-shadydom=true'
```

Comment:
Shouldn't this be:
```
WCT.loadSuites([
        'my-view1.html?dom=shadow',
        'ryr-location.html?dom=shadow',
        'my-view1.html?wc-shadydom=true',
        'ryr-location.html?wc-shadydom=true'
      ]);
```
Or something like that? The lines would be duplicated otherwise.